drivers/charger/Kconfig

```diff
@@ -19,8 +19,13 @@ config CHARGER_INIT_PRIORITY
 	help
 	  Battery charger initialization priority.
 
+config CHARGER_INIT_ENABLE_CHARGING
```

Comment:
This should be in devicetree rather than Kconfig

Reply:
Yeah I agree with that, will do